### PR TITLE
new 100m config files (used on Kisti)

### PR DESCRIPTION
**Pull request by jasonmotha**

### CODE REVIEW

File: templates/gmsim/20.4.1.1/root_defaults.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+bb:
+  fmin: 0.5
+  fmidbot: 1.0
+  no-lf-amp: True
+  dt: 0.005
+emod3d:
+   emod3d_version: 3.0.8
+flo: 1.0
+dt: 0.005
+pSA_periods: [0.01, 0.02, 0.03, 0.04, 0.05, 0.075, 0.1, 0.12, 0.15, 0.17, 0.2, 0.25, 0.3, 0.4, 0.5, 0.6, 0.7, 0.75, 0.8, 0.9, 1.0, 1.25, 1.5, 2.0, 2.5, 3.0, 4.0, 5.0, 6, 7.5, 10.0]
```
Comment (jpa198) [inline templates/gmsim/20.4.1.1/root_defaults.yaml:18]: Needs an im entry as per https://github.com/ucgmsim/slurm_gm_workflow/blob/master/templates/gmsim/20.4.1.2/root_defaults.yaml

**Review by jpa198 — CHANGES_REQUESTED** [review]

File: templates/gmsim/20.4.1.1/root_defaults.yaml (hunk 1 above)
Comment (jpa198) [reply templates/gmsim/20.4.1.1/root_defaults.yaml:28]: This line is now deprecated

**Review by jpa198 — APPROVED** [review]
Sure, looks ok
I do want to have an audit of the emod3d files at some point though. I know there a few that don't get used.

**Comment by jasonmotha** [comment]
The audit is a good idea. You are probably the person with the best idea of EMOD3D params.

**Comment by jpa198** [comment]
> The audit is a good idea. You are probably the person with the best idea of EMOD3D params.

Yeah, it should be a scoped ticket though, given how potentially breaking of a change it is. It would include set_runparams too, as that adds some junk too

**Review by shot0829 — APPROVED** [review]

**Pull request merged**